lib/YuiRestClient/App.pm

```diff
@@ -84,6 +85,14 @@ sub label {
     });
 }
 
+sub debug_label {
```

Comment:
I don't like our current naming style (I prefer cammel or something with underscore) in this file but let's follow the same approach and call it 'debuglabel'

Reply:
I got rid of this for now, as anyway it never worked.